tests/baseTestAPI.py

```diff
@@ -184,10 +181,7 @@ def setUp(self):
     def tearDownClass(cls):
         """Tear down class-level resources."""
         cls.interfaces.close()
-        dropDatabase(cls.interfaces.userDb.dbName)
-        dropDatabase(cls.interfaces.jobDb.dbName)
-        dropDatabase(cls.interfaces.errorDb.dbName)
-        dropDatabase(cls.interfaces.validationDb.dbName)
+        #dropDatabase(cls.interfaces.jobDb.dbName)
```

Comment:
Uncommented the drop statement.  As all the interfaces point at the same DB now, the drop here can be applied to any one of them.